src/app/countries/index.js

```diff
@@ -120,6 +129,18 @@ function getCountryCodeFromProposal (proposal: ProposalDTO): ?string {
   return proposal.serviceDefinition.locationOriginate.country
 }
 
+function getCountrySuccessRateFromProposal (proposal: ProposalDTO): number {
+  if (!proposal.metrics || !proposal.metrics.connectCount) {
+    return UNKNOWN_PROPOSAL_SUCCESS_RATE
+  }
+  const count = proposal.metrics.connectCount
+  const total = count.success + count.fail
+  if (total > 0) {
+    return count.success / (count.success + count.fail)
```

Comment:
You can re-use `total`
```suggestion
    return count.success / total
```